nautobot_golden_config/migrations/0025_auto_20230707_0921.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.16 on 2023-07-07 09:21
```

Comment:
It's a better practice to rename autogenerated filename into name of your choice

Reply:
renamed!